### ISSUE TITLE
SQL Keywords as column names

### ISSUE THREAD

**Issue by TapanJasthi-cpi** [jmoiron/sqlx#620]
So I'm looking to use `rows.StructScan(&structObj)` to scan a row into a my defined struct.
However, I'm running into issues and I'm suspecting that it's because one of my columns in the table is titled `"desc"`.
The struct I'm using is here:
```
type TableName struct {
	ID            int            `db:"id"`
	Name          sql.NullString `db:"name"`
	Desc          sql.NullString `db:"desc"`
	IsAutomatic   sql.NullInt64  `db:"is_automatic"`
}
```
The row was obtained using a select query: 
`SELECT id, name, 'desc', is_automatic FROM table_name WHERE id = 1275 LIMIT 1`
The values as JSON are `{id:1275, name:"clark", desc:"1234", is_automatic:0}`.

When I call `rows.StructScan(&structObj)`, the selected row should be scanned into the struct `structObj`. It's working as expected, except the value for desc in the struct is now the string `"desc"` when it's supposed it to be `"1234"`.

Basically, rows.StructScan is not scanning the column "desc" into the struct. Am I doing something wrong here or is sqlx not equipped to do `StructScan` with columns with SQL keywords. If it's the latter, would it be possible to add support for columns with special names like this?

**Comment by The-Phantom-Of-The-Opera** [comment]
try to use db.Get(&structObj, sql) or db.Select(&[]objs, sql) method?

**Comment by TapanJasthi-cpi** [comment]
Ended up solving my issue. It was a combination of a poorly formed struct tag and using a pointer of a pointer by mistake.
